.github/workflows/linters.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: "12.x"
+          node-version: "18.x"
```

Comment:
[Hint] you can fix the lighthouse error by changing the node version to `12.x` again and hopefully, it will pass.

Reply:
Thank you so much for your feedback. I shall make the changes and will again ask for review